src/Microsoft.DotNet.ScenarioTests.Common/Program.cs

```diff
@@ -92,7 +92,7 @@ public static async Task<int> Main(string[] args)
 
             rootCommand.SetAction((ParseResult parseResult) =>
             {
-                Invoke(parseResult.GetValue(dotnetRootOption)!,
+                return Invoke(parseResult.GetValue(dotnetRootOption)!,
```

Comment:
Should these `!` be `?? throw new ArgumentException(...)`?

Reply:
Technically, I don't think so. The parameters are required or have defaults. But it couldn't hurt.

Reply:
QQ: is there a better help available? E.g. `Expected {dotnetRootOption.Prototype} to be set`? I know that in `MonoOptions` you can get the name and tell the user what's the real option one needs

Reply:
It won't even make far enough to throw ArgumentException though. You may as well throw InvalidOperationException. If you don't pass it, the parser will fail out before it even hit this point, and will provide appropriate help.

Reply:
The issue is that the API always returns a nullable string even if the value is guaranteed to be non-null and a valid directory (see above where the validator ensures that the file "dotnet.exe/dotnet" exists in the directory passed as dotnet-root.